README.md

```diff
@@ -76,7 +76,7 @@ OPTIONS:
 
 SUBCOMMANDS:
     config      Configuration profile commands
-    core        Core schema commands
+    supergraph        Supergraph schema commands
```

Comment:
mmmm this tabbed out kinda weird it looks like, did you rerun the help command and repaste or just do a find/replace?